upload/upload/gen-secret-key.py

```diff
@@ -0,0 +1,10 @@
+import os
```

Comment:
hmm. I think we could actually include secret creation in the yaml file.

Secrets look like:
```
apiVersion: v1
kind: Secret
metadata:
  name: mysecret
type: Opaque
data:
  username: YWRtaW4=
  password: MWYyZDFlMmU2N2Rm
```

where the data fields are base64 encoded.

`redeploy` just calls `gen-secret-key.py` to make the secret and slams it into the yaml.

It means the secret changes with each deploy :/ but we have less latent state to manage :).

Reply:
So if you want to share a session login between multiple services in the Hail "suite", then we need a single common secret key that they all use.

Reply:
This would also invalidate sessions between redeploys which seems wrong.

Reply:
A common secret key seems achievable. Agreed redeploys in this setting invalidate the session. Not the worst thing since we're talking with a new server? At some point we will have to address these issues to get redeployable k8s.

Reply:
There might be some confusion.  I ran this code once and added the generated secret key as a k8s secret.  There is no session invalidation problem.  I just checked it in here for reference in case we need to generate generate another one (e.g. rebuild k8s).